.github/pull_request_template.md

```diff
@@ -1,14 +1,37 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+
+1. If this is your first time, take a look at our developer guide (TODO link dev guide)!
+2. Make sure to check these marks:
+-->
+* [ ] I have read [CONTRIBUTING.md](../CONTRIBUTING.md) and [CODE_OF_CONDUCT.md](../CODE_OF_CONDUCT.md)
+* [ ] I have added to [CHANGELOG.md](../CHANGELOG.md) relevant changes and any other user facing change
+* [ ] I have added or ran the appropriate tests
+<!--
+3. If the PR is unfinished, mark it as `[WIP]` in the title
+-->
+
+#### What this PR does / why we need it:
+
+#### Which issue(s) this PR fixes:
+
 <!--
+*Automatically closes linked issue when PR is merged.
+Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
+_If there is no related issue, do not use `Fixes`_*
+-->
+Fixes #
 
-**Please, carefully describe what the PR does and why you are opening it.**
+#### Special notes for your reviewer:
 
-Short check list:
+#### Does this PR introduce a user-facing change?
+* [ ] Yes
+* [ ] No
```

Comment:
is this not redundant with current line 7?

Reply:
No, this one is meant to be read by the reviewer when skimming the PR, while line 7 is meant for the contributor to make sure everything was done right

Reply:
Oh alright! Then I suggest we indent reviewer notes one more layer (`#####`) to make it clearer that they're children of special reviewer notes